browser/src/control/Control.TopToolbar.js

```diff
@@ -226,14 +227,123 @@ class TopToolbar extends JSDialog.Toolbar {
 		}
 	}
 
+	createOverflowMenu() {
```

Comment:
nice, would be also good to use our helper for popups: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Util.Dropdown.js#L22

example usage: https://github.com/CollaboraOnline/online/blob/c70220560a586391c900a7d83a638370eafe9e14/browser/src/control/Control.UserList.ts#L220